drivers/bluetooth/hci/Kconfig

```diff
@@ -136,8 +136,18 @@ config BT_SILABS_EFR32
 	help
 	  Use Silicon Labs binary Bluetooth library to connect to the
 	  controller.
+
 source "drivers/bluetooth/hci/Kconfig.silabs"
 
+config BT_SILABS_SIWX91X
```

Comment:
This should probably go before the `source "drivers/bluetooth/hci/Kconfig.silabs"` line, so that options specific to the 917 HCI driver can be put into `Kconfig.silabs`

Reply:
Actually, I think it'd be even justified to move the top-level Silabs driver options into that file too, however if the current situation is the common practice for HCI drivers we can do the refactoring in a later PR too.